content/zh-cn/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 官方 CVE 订阅源
+weight: 25
+outputs:
+  - json
+  - html 
+layout: cve-feed
+---
+<!--
+title: Official CVE Feed
+weight: 25
+outputs:
+  - json
+  - html 
+layout: cve-feed
+-->
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+<!--
+This is a community maintained list of official CVEs announced by
+the Kubernetes Security Response Committee. See
+[Kubernetes Security and Disclosure Information](/docs/reference/issues-security/security/)
+for more details.
+
+The Kubernetes project publishes a programmatically accessible
+[JSON Feed](/docs/reference/issues-security/official-cve-feed/index.json) of 
+published security issues. You can access it by executing the following command:
+-->
+这是由 Kubernetes 安全响应委员会（Security Response Committee, SRC）公布的经社区维护的官方 CVE 列表。
+更多细节请参阅 [Kubernetes 安全和信息披露](/zh-cn/docs/reference/issues-security/security/)。
+
+Kubernetes 项目就已发布的安全问题发布了一个可使用程序访问的
+[JSON 订阅源](/docs/reference/issues-security/official-cve-feed/index.json)。
```

Comment:
I think the Chinese for “[JSON Feed](https://www.jsonfeed.org/)” might be “JSON Feed”.